packages/repository/src/__tests__/unit/repositories/crud.repository.unit.ts

```diff
@@ -86,6 +85,10 @@ class CrudConnectorStub implements CrudConnector {
   ): Promise<Count> {
     return Promise.resolve({count: this.entities.length});
   }
+
+  beginTransaction(options: Options, cb: Callback) {
```

Comment:
Do we allow Promise here?

Reply:
Not yet, connectors must use callbacks now. See https://github.com/strongloop/loopback-datasource-juggler/issues/1659

Let's add a code comment here to explain & include a link to that GH issue.